bolides/sdf.py

```diff
@@ -0,0 +1,260 @@
+import pandas as pd
+import requests
+import numpy as np
+from . import ROOT_PATH
+
+
+class ShowerDataFrame(pd.DataFrame):
```

Comment:
I do not have time to test this functionality. I'll just give it a pass and assume it works. It's not central to the main functionality of the bolide package.